Rec: prep for 4.5.0 beta1 (#10213)
PowerDNS/pdns: pdns/recursordist/docs/changelog/4.5.rst

Changelogs for 4.5.X
====================
.. changelog::
  :version: 4.5.0-beta1
  :released: 26th of March 2021

  .. change::
    :tags: Improvements
    :pullreq: 9995
    :tickets: 7982

>     Support tcp fastopen connect.

Reviewer: Perhaps we could make it more clear than it is for outgoing connections, something like:
```suggestion
    Support tcp fastopen connect on outgoing connections.
```

  .. change::
    :tags: Improvements
    :pullreq: 8918

    Implement EDNS0 padding (rfc7830) for outgoing responses.

  .. change::
    :tags: Improvements
    :pullreq: 10057

>     Get rid of early zone cut computation.

Reviewer: ```suggestion
    Get rid of early zone cut computation when doing DNSSEC validation.
```

  .. change::
    :tags: Improvements
    :pullreq: 10182
    :tickets: 10177

    Insert hints as non-auth into cache.

  .. change::
    :tags: Bug Fixes
    :pullreq: 10185

    Make sure we take the right minimum for the packet cache TTL data.

  .. change::
    :tags: Improvements
    :pullreq: 10178
    :tickets: 10125

    Don't pick up random root NS records from AUTHORITY sections.

  .. change::
    :tags: Improvements
    :pullreq: 10161
    :tickets: 7591

>     DATA size is unreliable, start using RES, as it seems reliable and relevant.

Reviewer 2: This could use some words for those without context

>     DATA size is unreliable, start using RES, as it seems reliable and relevant.

Reviewer: ```suggestion
    Using DATA to report memory usage is unreliable, start using RES instead as it seems reliable and relevant.
```
